src/python/pants/backend/codegen/protobuf/java/java_protobuf_library.py

```diff
@@ -18,29 +19,33 @@
 class JavaProtobufLibrary(ImportJarsMixin, JvmTarget):
   """A Java library generated from Protocol Buffer IDL files."""
 
+  imported_target_kwargs_field = 'imports'
+  imported_target_payload_field = 'import_specs'
+
   def __init__(self, payload=None, buildflags=None, imports=None, **kwargs):
     """
     :param buildflags: Unused, and will be removed in a future release.
     :param list imports: List of addresses of `jar_library <#jar_library>`_
       targets which contain .proto definitions.
     """
     payload = payload or Payload()
+    deprecated_conditional(
```

Comment:
Including comments next to these that reference the ticket you created would be great. But not worth another CI burn.

Reply:
Done, as CI was failing mysteriously anyway. If it continues to do so I will look into why.